ocaml/auth/extauth_plugin_ADpbis.ml

```diff
@@ -30,21 +42,8 @@ struct
 
   let user_friendly_error_msg = "The Active Directory Plug-in could not complete the command. Additional information in the logs."
 
-  open Stdext.Xstringext
```

Comment:
Yes, but I mean it is duplicated in master too right now (https://github.com/xapi-project/xen-api/blob/master/ocaml/xapi/extauth_plugin_ADpbis.ml), just as you have corrected here.